mail_embed_image/tests/test_mail_embed_image.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from base64 import b64encode
+from odoo.tests import common
+from mock import patch, mock
+from lxml import html
+from requests import get
+from odoo import api, registry, SUPERUSER_ID
+
+
+reg = registry(common.get_db_name())
+env = api.Environment(reg.cursor(), SUPERUSER_ID, {})
+MockBuildEmail = mock.Mock(wraps=env['ir.mail_server'].build_email)
+MockSendEmail = mock.Mock(wraps=env['ir.mail_server'].send_email)
+
+
+class TestMailEmbedImage(common.TransactionCase):
+
+    post_install = True
+    at_install = False
```

Comment:
don't, there's no reason to do this and it can fall on your feet if some other module later does stuff you don't expect